jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -51,6 +51,99 @@ typedef ecma_value_t (*ecma_builtin_dispatch_routine_t)(uint16_t builtin_routine
  */
 typedef ecma_value_t (*ecma_builtin_dispatch_call_t)(const ecma_value_t arguments_list[],
                                                      ecma_length_t arguments_number);
+/**
+ * Definition of a builtin descriptor which contains the builtin object's:
+ * - prototype objects's id (12-bits)
+ * - type (3-bits)
```

Comment:
These are only 15 bits. Is the 16th bit unused?

Reply:
Oops! Good catch! All the remaining bits(13) are used for the prototype objects's id.